site/content/docs/standard/content-styles/hover-effects/index.html

```diff
@@ -50,6 +50,32 @@
   {{< /twsnippet/demo >}}
 
 
+  <!-- Section: Note about usage with animation  -->
+  <section>
+    <h3>Showing the animation effect</h3>
+    <p class="mb-3">
+      As you can see in the above example, you can add any animation to any
+      element on the page by using the following classes:
+    </p>
+    <ul>
+      <li>
+        <code>transition</code> : Used to animate the chaged properties on
+        hover.
+      </li>
+
+      <li>
+        <code>duration-x</code> : Used to define the duration of the animation
+        where <code>x</code> is the time in melli seconds; ex:
+        <code>duration-300</code>.
+      </li>
+    </ul>
+    <p>
+      then, you should be able to add any type of animation by changing the
+      property on hover by using the class: <code>hover:tw-class-name</code> ex:
```

Comment:
Please change `ex` to `e.g.`. Also please change `hover:tw-class-name` to `hover:[tw-class-name]` to make sure users will not use the actual class name from this example.

Reply:
good one 👌🏻